### ISSUE TITLE
SPIKE: Research Internal Protocol Error

### ISSUE THREAD

**Issue by marriej** [department-of-veterans-affairs/va.gov-team#7745]
The following Internal Protocol error has been observed in the staging.va.gov environment. At various times and on several browsers. For now we have not been able to replicate this error on demand. This ticket is to look into ways to reproduce this error consistently so the IAM team can investigate further.

![Screen Shot 2020-03-31 at 6.08.50 PM.png](https://images.zenhubusercontent.com/5e5d21ccbf668393ce8d133e/ec849909-2e21-48b0-8173-8a8316d056c8)

**Comment by ericbuckley** [comment]
I was able to replicate the "Internal Protocol Error" on 4/7 around 6:16pm eastern.

Browser: Firefox 74.0.1
OS: Mac 10.15.3
Environment: https://staging.va.gov/
User: vets.gov.user+262@gmail.com

1) opened the preferences window and deleted all cache/cookies associated to any *.va.gov domains.
2) went to https://staging.va.gov/ and and used the signin modal to authenticate with ID.me credentials
3) entered the username/password for vets.gov.user+262@gmail.com and clicked through the continue dialogs
4) was SUCCESSFULLY redirected back to sqa.eauth.va.gov, then to staging.va.gov
5) waited 30 mins to be automatically logged out
6) clicked the signin button again to launch the modal and authenticate with the same ID.me credentials
7) after entering the credentials and clicking through the continue dialogs, I landed on https://sqa.eauth.va.gov/isam/sps/auth?PartnerId=https://ssoe-sp-staging.va.gov with the "Internal Protocol Error"

Sent an email to the IAM team with the above details.  Will wait to hear from them before I do any more debugging.
